pkg/shell/shell.html

```diff
@@ -1529,6 +1544,58 @@ <h4 class="modal-title" id="account-confirm-delete-title"></h4>
       </div>
     </div>
 
+    <script id="authorized-keys-tmpl" type="x-template/mustache">
+      {{#keys}}
+        <tr>
+        <td>{{comment}}</td>
+        <td class="fingerprint">
+           {{fp}}{{^valid}}<span translatable="yes">Invalid key</span>{{/valid}}
+        </td>
+        <td class="cell-buttons"><button data-raw="{{raw}}" class="btn btn-default accounts-self-privileged account-remove-key">-</button></td>
```

Comment:
The delete button should have a hover-text to explain what it does.
@andreasn This is a destructive action. Should it be red or somehow protected? Containers have the two-step process and the red trashcan icon. The list has a cursor change on hover that implies it can be clicked, yet the only thing I can click on that has an effect is the delete button...